helm-charts/charts/kube-starrocks/charts/starrocks/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if and (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") .Values.metrics.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "starrockscluster.name" . }}-fe
+  namespace: {{ template "starrockscluster.namespace" . }}
+  labels:
+    cluster: {{ template "starrockscluster.name" . }}
+spec:
+  endpoints:
+    - port: http
```

Comment:
does this mean that cr controller will search for the correct port from the given port name of the service?

Reply:
yes